### PR TITLE
2.5.0.2 release notes

### PR DESCRIPTION
**Pull request by harsh-px**
Signed-off-by: Harsh Desai <harsh@portworx.com>

### CODE REVIEW

File: content/reference/release-notes/portworx.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ description: Notes on Portworx releases.
...
 ---
+## 2.5.0.2
+
+June 12, 2020
+
+## Fixes
+
+The following issues have been fixed:
+
+|**Issue Number**|**Issue Description**|
```
Comment (vilasd) [inline content/reference/release-notes/portworx.md:16]: Can we try to follow this format if possible

Issue Description:

User Impact: When the image digest was not available, OCI-Monitor would not detect manually uploaded Portworx images, and would attempt to pull the Portworx image, potentially failing in air-gapped environments.

Resolution: Portworx now has improved image detection, even in cases where image digests are not available.

File: content/reference/release-notes/portworx.md [hunk 2]
```diff
@@ -5,6 +5,18 @@ description: Notes on Portworx releases.
...
+|----|----|
+|PWX-9401|Fix parsing of group and group enforcement values during volume creation to workaround [this issue in Kubernetes 1.13.5 and lower](https://github.com/kubernetes/kubernetes/issues/76340).|
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md:18]: @harsh-px These tickets are pretty light in information. Can we put these fixes into our detailed ‘description, user impact, and resolution’ format? I took a stab at this one, not sure if I got it right: 

```suggestion
|PWX-9401| A [bug in Kubernetes 1.13.5 and lower](https://github.com/kubernetes/kubernetes/issues/76340) caused the Portworx volume driver to occasionally save annotations from one PVC into the parameters for another, crashing the Kubernetes controller manager.<br/><br/>**User impact:** Users saw volumes in Portworx that were not bound in Kubernetes.<br/><br/>**Resolution:**  Portworx now parses group and group enforcement values differently during volume creation to workaround this Kubernetes bug.|
```

File: content/reference/release-notes/portworx.md (hunk 1 above)
Comment (harsh-px) [reply content/reference/release-notes/portworx.md:16]: done

File: content/reference/release-notes/portworx.md (hunk 2 above)
Comment (harsh-px) [reply content/reference/release-notes/portworx.md]: Update as per Vilas's suggestion. Can you re-review?

**Comment by harsh-px** [comment]
@adityadani we might need to update the volume update doc

https://docs.portworx.com/reference/cli/updating-volumes/

File: content/reference/release-notes/portworx.md [hunk 3]
```diff
@@ -5,6 +5,18 @@ description: Notes on Portworx releases.
...
+|PWX-10207| Allow overriding a volume's group field using pxctl.|
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md]: After talking with Aditya, I think this one is an improvement and not a bug fix. If so, we don't need to mention all the details, but we should move it under and improvements section. 

```suggestion
### Improvements

Portworx has upgraded or enhanced functionality in the following areas:

| **Improvement Number** | **Improvement Description** |
|----|----|
|PWX-10207| You can now override a volume's group field using pxctl: <NEED COMMAND>|
```

Comment (harsh-px) [reply content/reference/release-notes/portworx.md]: Moved to improvements section. Check now.

File: content/reference/release-notes/portworx.md [hunk 4]
```diff
@@ -5,6 +5,18 @@ description: Notes on Portworx releases.
...
+
+June 12, 2020
+
+## Fixes
+
+The following issues have been fixed:
+
+|**Issue Number**|**Issue Description**|
+|----|----|
+|PWX-9401|Fix parsing of group and group enforcement values during PVC creation. <br/><br/>**User Impact:** Due to [this issue in Kubernetes 1.13.5 and lower](https://github.com/kubernetes/kubernetes/issues/76340), a PVC can get created with a different group ID than what is set in it's annotations. <br/><br/>**Resolution:** Portworx now uses the group value from the PVC annotation that's fetched at runtime from the Kubernetes API during volume creation.|
```
Comment (nathan-px) [inline content/reference/release-notes/portworx.md:18]: I'm still not following the issue description. It seems like "Fix parsing of group and group enforcement values during PVC creation." is more appropriate for the "resolution" section. How about this?

```suggestion
|PWX-9401| A [bug in Kubernetes 1.13.5 and lower](https://github.com/kubernetes/kubernetes/issues/76340) caused the Portworx volume driver to occasionally save annotations from one PVC into the parameters for another. <br/><br/>**User Impact:** Portworx may have created a PVC with a different group ID than the one set in its annotations. <br/><br/>**Resolution:** Portworx now uses the group value from the PVC annotation that's fetched at runtime from the Kubernetes API during volume creation to ensure the group ID doesn't change.|
```

Comment (harsh-px) [reply content/reference/release-notes/portworx.md:18]: Shouldn't the first line of release note talk about the fix in one line?

`A bug in Kubernetes caused the Portworx volume driver to occasionally save annotations from one PVC into the parameters for another.` just describes the problem.

I'm okay either ways and will follow your standard. Just brainstorming :)

Comment (nathan-px) [reply content/reference/release-notes/portworx.md:18]: Our standard is currently to describe the issue in the first line, how it impacts the users in the second, and then the fix in the third. 

I do see your point though. Let's stick with the standard until we make any deliberate decision to deviate from it.

**Pull request merged**